plugin/output/splunk/splunk_test.go

```diff
@@ -0,0 +1,66 @@
+package splunk
+
+import (
+	"github.com/ozonru/file.d/pipeline"
+	insaneJSON "github.com/vitkovskii/insane-json"
+	"go.uber.org/zap"
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+)
+
+func TestEventKey(t *testing.T) {
+	testsCases := map[string]string{
+		`{"msg":"AAAA","some_field":"BBBB"}`:         `{"event":{"msg":"AAAA"},"some_field":"BBBB"}`,
+		`{"msg":"AAAA","some_field":{"boo": "bar"}}`: `{"event":{"msg":"AAAA"},"some_field":{"boo":"bar"}}`,
+	}
+
+	for k, v := range testsCases {
+		if resp := testOne(k, t); resp != v {
+			println(k)
+			println(v)
+			t.Fatal(resp)
```

Comment:
Although `t.Fatal` is correct, we prefer use `assert` from "github.com/stretchr/testify/assert" in this repo. 
```go
assert.Equal(t, v, resp, "wrong resp for k: %s, v: %s", k, v)
```

Reply:
Done